README-QUIC.md

```diff
@@ -0,0 +1,25 @@
+# QUIC
+
+In addition to an OpenSSL-compatible libcrypto implementation,
```

Comment:
We don't provide the OpenSSL implementation; it was stripped out. (Unlike the previous `quictls/openssl` releases)

Reply:
I have added a clarification that libcrypto API == cryptographic primitives only, (i.e., EVP-stuff, EC curves etc) and does not include OpenSSL's 3.x QUIC transport.

It seems that picoquic and mvfst  use libcrypto for low-level primitives and their own TLS implementations.

TL;DR 
```
To be clear: the libcrypto API referred to here is the set of cryptographic primitives in OpenSSL's libcrypto and
not the QUIC transport feature (of OpenSSL 3.x). QUIC implementations that only use cryptographic primitives from libcrypto typically
require a separate SSL/TLS helper library.
```